package.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "smart-underline",
   "version": "v4.2.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/EagerIO/SmartUnderline.git"
+  },
+  "keywords": [
+    "smart",
+    "underline",
+    "Readable",
```

Comment:
I think lowercase would be more consistent with the other keywords, so yes, please change.